core/embed/lib/gfx_draw.h

```diff
@@ -141,22 +144,26 @@ void gfx_draw_bar(gfx_rect_t rect, gfx_color_t color);
 // Not all bitmap formats are supported now. Please see the implementation.
 void gfx_draw_bitmap(gfx_rect_t rect, const gfx_bitmap_t* bitmap);
 
+#define GFX_ALIGN_LEFT 0x00    // Align text to the left (horizontal)
```

Comment:
may be better as enum

Reply:
yes, it's better, fixed, see 4c1cd9be909859df36e3e62711dd69ebd25b3c3f